### ISSUE TITLE
Hide Close Buttons

### ISSUE THREAD

**Issue by GreenLunar** [QupZilla/qupzilla#1883]
Apparently, it is a valuable feature to some users.

> In version 31, the `browser.tabs.closeButtons` option was removed. This restartless addon forces what previously used to be a value of 2 for that option: no close buttons on any tabs.

From: https://github.com/djc/no-close-buttons

**Comment by innermous** [comment]
@GreenLunar You still can use "Show close buttons on inactive tabs"="Never".

**Comment by GreenLunar** [comment]
Indeed, but there is not yet an option to hide close button of active tab.
